Makefile

```diff
@@ -147,7 +147,7 @@ copyright-check:
 
 .PHONY: lint
 lint:
-	black $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
+	black $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py --exclude $(OUTPUT_FOLDER)
```

Comment:
Also exclude output folder from isort

Reply:
Excluded in `setup.cfg`

Reply:
Black doesn't support setup.cfg, that's why left here as the command line parameter.